src/hooks/useQuery.test.tsx

```diff
@@ -158,7 +164,21 @@ describe('on change', () => {
 describe('execute query', () => {
   it('triggers query execution', () => {
     renderer.create(<QueryUser {...props} />);
-    act(() => execute());
+    act(() => execute!());
```

Comment:
Non-nullary operator for tests, since `execute` can, from a type perspective, be `undefined` in the course of running these tests.